nahusenayElias/Team_4
# DRD-100: Front page styling

## Related ticket
[DRD-100](https://edu-team4-react24k.atlassian.net/jira/software/projects/DRD/boards/1?selectedIssue=DRD-100)

## In this PR: 
- Added content to the frontpage in Drupal
- Centered text and images on the frontpage, increased font size
- Changed project case link color from blue to orange 
- I also did some styling to the Contact page in this PR and added a button
- Added padding-top to the header Team 4 next to the logo so that they are in line with each other
- Added margin to the Blog h1 so that it is entirely visible

## Testing instructions
- checkout branch
- lando drush cim
- lando db-import DRD-100.sql.gz
- Check the front page that you see the updated version
- Check the Contact page too, test buttons

NOTE!  I had to add one image manually to the front page because I couldn't/didn't have time to figure out how to fetch it from Drupal. 😄 

[DRD-100]: https://edu-team4-react24k.atlassian.net/browse/DRD-100?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
